app/__init__.py

```diff
@@ -1,12 +1,12 @@
 from flask import Flask
-from .db import db, migrate
-from .routes.planet_routes import planets_bp
+from app.db import db, migrate
+from app.routes.planet_routes import planets_bp
 
 def create_app(test_config=None):
     app = Flask(__name__)
 
-    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
-    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://postgres:postgres@localhost:5432/solar_system_development'
+    app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+    app.config["SQLALCHEMY_DATABASE_URI"] = "postgresql+psycopg2://postgres:postgres@localhost:5432/solar_system_development"
```

Comment:
I see.. switched to double quotes. Both single and double quotes work in Python right?